### PR TITLE
Change warning message

### PR DESCRIPTION
**Pull request by cqc-alec**
Test fails with:

```
self = <pytket.qir.conversion.conversion.QirGenerator object at 0x7fc3ec940c70>, command = PhasedX(3.5, 0.5) node[0];

    def _rebase_command_to_gateset(self, command: Command) -> Optional[Circuit]:
        """Rebase to the target gateset if needed."""
        optype = command.op.type
        params = command.op.params
        args = command.args
        if optype not in self.module.gateset.base_gateset:
            circuit = Circuit(self.circuit.n_qubits, self.circuit.n_bits)
>           circuit.add_gate(optype, params, args)
E           RuntimeError: Circuit does not contain unit with id: node[0]

../env/lib/python3.10/site-packages/pytket/qir/conversion/conversion.py:292: RuntimeError
```

### CODE REVIEW

**Review by cqc-melf — DISMISSED** [review]

**Comment by cqc-alec** [comment]
Ah I made a mistake (ran the wrong test!). Getting an error from `if "pytket_to_qir" not in locals()`.
Closing for now...

File: pytket/extensions/quantinuum/backends/quantinuum.py [hunk 1]
```diff
@@ -632,16 +632,15 @@ def process_circuits(
             else:
                 assert language == Language.QIR
                 warnings.warn(
-                    "Support for Language.QIR is experimental; this will probably fail!"
+                    "Support for Language.QIR is experimental; this may fail!"
                 )
-                if "pytket_to_qir" not in locals():
+                try:
```
Comment (cqc-melf) [inline pytket/extensions/quantinuum/backends/quantinuum.py:637]: Do we want to add a check of the version of pytket-qir here as well?

Comment (cqc-alec) [reply pytket/extensions/quantinuum/backends/quantinuum.py:637]: > Do we want to add a check of the version of pytket-qir here as well?

What should the check be?

Comment (cqc-melf) [reply pytket/extensions/quantinuum/backends/quantinuum.py:638]:
```suggestion
                try:
                    import pytket.qir.__extension_version__ as version
                    if (int(version.split(".")[1]) < 2):
                        raise Error("please update to pytket-qir >= 0.2.0")

Comment (cqc-melf) [reply pytket/extensions/quantinuum/backends/quantinuum.py:638]: Something like this?

Comment (cqc-melf) [reply pytket/extensions/quantinuum/backends/quantinuum.py:638]: Yes I had something in mind like this.

**Review by cqc-melf — COMMENTED** [review]
I have put out a release candidate 0.2.0rc7 with the newest changes.

**Review by cqc-melf — APPROVED** [review]

**Pull request merged**